webapp/tests/karma/unit/services/relative-date.js

```diff
@@ -223,6 +210,6 @@ describe('RelativeDate Service', () => {
     chai.expect(formatDateRelativeTime.callCount).to.equal(3);
     chai.expect(formatDateTime.callCount).to.equal(1);
     chai.expect(formatDateRelativeDay.callCount).to.equal(1);
-    done();
   });
+
```

Comment:
Consider adding unit tests for the new `getRelativeDate` exported function.

Reply:
Thank you! Done.